UnitySDK/Assets/ML-Agents/Examples/SharedAssets/Scripts/SettingsOverrides.cs

```diff
@@ -0,0 +1,52 @@
+using UnityEngine;
+using MLAgents;
+
+public class SettingsOverrides : MonoBehaviour
+{
+    // Original values
+    Vector3 m_OriginalGravity;
+    float m_OriginalFixedDeltaTime;
+    float m_OriginalMaximumDeltaTime;
+    int m_OriginalSolverIterations;
+    int m_OriginalSolverVelocityIterations;
+
+    public float gravityMultiplier = 1.0f;
```

Comment:
Not sure if we should have gravity multiplier or gravity setting (with default at -9.81)

Reply:
All of the scenes today use a multiplier (but FloatPropertiesChannel use the negative of the value)